src/odemis/model/_components.py

```diff
@@ -107,12 +107,16 @@ class Component(ComponentBase):
     Component to be shared remotely
     '''
 
-    def __init__(self, name, parent=None, children=None, daemon=None):
+    def __init__(self, name, parent=None, children=None, dependencies=None, daemon=None):
         """
         name (string): unique name used to identify the component
         parent (Component): the parent of this component, that will be in .parent
         children (dict str -> Component): the children of this component, that will
-            be in .children. Objects not instance of Component are skipped
```

Comment:
I don't think this is correct. The idea of the "children" argument is that if the component has already created the children, it can directly pass them, and the VA will be initialised with them already.
For the microscope class, it works as a weird hack, but let's not mention it here. Only in the Microscope class.